public/app/core/components/NavLandingPage/NavLandingPage.tsx

```diff
@@ -15,43 +13,23 @@ interface Props {
 
 export function NavLandingPage({ navId, header }: Props) {
   const { node } = useNavModel(navId);
-  const styles = useStyles2(getStyles);
   const children = node.children?.filter((child) => !child.hideFromTabs);
-
   return (
     <Page navId={node.id}>
       <Page.Contents>
-        <div className={styles.content}>
+        <Flex direction="column" gap={2}>
           {header}
           {children && children.length > 0 && (
-            <section className={styles.grid}>
+            <Grid display="grid" gap={3} templateColumns="repeat(auto-fill, minmax(300px,1fr))" autoRows="138px">
               {children?.map((child) => (
-                <NavLandingPageCard
-                  key={child.id}
-                  description={child.subTitle}
-                  text={child.text}
-                  url={child.url ?? ''}
-                />
+                <GridItem key={child.id} displayContents>
```

Comment:
I'm not sure I understand the `displayContents` prop and also I don't think we need the `GridItem` if that is the only prop that is set right? I think we can remove it from all these examples you created unless it needs to specify which row/column it is rendered at?

Reply:
As I understand it, `display: contents` makes the children of the grid items be considered also grid and therefore apply the inheritance. I'll change the examples.